paddle/fluid/inference/tensorrt/convert/test_pool2d_op.cc

```diff
@@ -40,11 +40,13 @@ TEST(Pool2dOpConverter, main) {
   std::vector<int> strides({2, 2});
   std::vector<int> paddings({0, 0});
   std::string pooling_t = "max";
+  bool global_pooling = false;
```

Comment:
any need to test both true and false?

Reply:
we don't need the care the test attr, we always in the test mode.

Reply:
The test for global_pooling (true) is needed.

Reply:
Done!